### ISSUE TITLE
Arrow keys cannot be used to navigate search results

### ISSUE THREAD

**Issue by AdamLearns** [laurent22/joplin#9553]
### Operating system

macOS

### Joplin version

Joplin 2.13.9 (prod, darwin)

### Desktop version info

```
Joplin 2.13.9 (prod, darwin)

Client ID: 76122f1042cc4756b47308502b4b719d
Sync Version: 3
Profile Version: 44
Keychain Supported: No

Revision: 1bbec44
```

### Current behaviour

1. Start a search by pressing ⌘⇧F.
2. Type a search query that appears in multiple notes (e.g. "the").
3. Try pressing the down arrow on your keyboard to go to the search results. 

[Here's a video recording of me trying this out](https://clips.twitch.tv/ConsiderateCrowdedBoarSwiftRage-XDKNLJOTZtSAkZia).

### Expected behaviour

I expected the focus to go down to the search results and optionally to preview the selected note (although I don't mind pressing enter at that point either).

I did see issue #6352, but it seems to have been closed with no resolution.

**Comment by AdamLearns** [comment]
I downloaded the latest version to test this (`2.13.15`), and it's still an issue. I'm no longer using Joplin, so I probably won't be commenting in the future to keep this issue alive. 😢

**Comment by personalizedrefrigerator** [comment]
For reference, here are a few relevant files: 
- [NoteListControls.tsx](https://github.com/laurent22/joplin/blob/dd04a39f3b0add299adc0bf26e74c68813414ce3/packages/app-desktop/gui/NoteListControls/NoteListControls.tsx#L238)
- [SearchBar.tsx](https://github.com/laurent22/joplin/blob/dd04a39f3b0add299adc0bf26e74c68813414ce3/packages/app-desktop/gui/SearchBar/SearchBar.tsx#L30)

**Comment by laurent22** [comment]
Before working on this issue we should define how it should work. I've never seen an app move to the search results by pressing the down arrow or even Enter. Tab maybe? Because it won't be documented it needs to be intuitive

**Comment by njmulsqb** [comment]
I guess Enter makes the most sense because that's the traditional approach towards search i.e. when we talk about search engines. Enter takes us to the search results.

**Comment by laurent22** [comment]
> I guess Enter makes the most sense because that's the traditional approach towards search i.e. when we talk about search engines. Enter takes us to the search results.

But here the results are displayed in realtime

**Comment by njmulsqb** [comment]
> But here the results are displayed in realtime

Right. I hadn't given it much thought just that I saw this issue open labelled "good-first-issue" I assumed this functionality is what Joplin is looking to implement.

**Comment by laurent22** [comment]
Yes, I think the problem is that I misread the task. I thought the suggestion was to use arrow keys to go from one result to another, but that's already implemented. Having a key to jump to the results from the search field is a reasonable feature to add too, but we need to clearly specify what we want

**Comment by G0maa** [comment]
Hello, I found this issue since it has `good first issue` label, I also read the discussion.

By chance, I noticed `discource` has the same feature (not quite), you press enter, query results, then you're able to navigate results if you press arrow-down. You can also navigate to the search bar using same arrows.

https://github.com/laurent22/joplin/assets/12874707/320b5bcb-2d2c-4b7b-b668-2c0b77d41d50

Edit: just cropped the video to make it more presentable.

**Comment by Varun-2538** [comment]
hey i have just noticed this issue and I think I can solve this , is the issue still open to work ?

**Comment by ahsanfarooq210** [comment]
Can we implement it using refs, in the `NoteListWrapper.tsx` file, we can pass a ref in the `NoteList2` component and set it there and we also pass it in the `SearchBar` component via `NoteListControls` component and we can shift the focus on the press of arrow down key in the `SearchBar` component.

**Comment by zeul22** [comment]
I would love to claim this. I might solve this issue.

**Comment by abhinav26966** [comment]
Hi, I'm interested in working on this issue and would love to contribute to the project. Could you please assign it to me? @laurent22 @personalizedrefrigerator

**Comment by shah0108** [comment]
Has this issue been solved?

**Comment by laurent22** [comment]
Hello, I am locking this issue as it attracts too many unhelpful comments.

- Yes, you can work on it
- No we won't assign the issue
- Yes the issue is still active
